src/sql/workbench/api/common/extHostModelViewDialog.ts

```diff
@@ -797,7 +796,11 @@ export class ExtHostModelViewDialog implements ExtHostModelViewDialogShape {
 		});
 	}
 
-	public updateButton(button: azdata.window.Button): void {
+	public updateButton(button: azdata.window.Button, isCustomButton: boolean = false): void {
```

Comment:
I guess I'd rather the param be called something more generic like `forceSecondary` or something - but I'll leave that up to you.